package.json

```diff
@@ -46,24 +43,20 @@
     "react-dropzone": "^11.3.4",
     "react-helmet": "^6.0.0",
     "react-hook-form": "^6.7.2",
-    "react-hotkeys": "^2.0.0",
     "react-hotkeys-hook": "^2.4.0",
     "react-json-view": "^1.19.1",
     "react-multi-select-component": "^2.0.12",
     "react-popper": "^2.2.5",
     "react-select": "^4.1.0",
     "react-toast-notifications": "^2.4.0",
     "react-tooltip": "^4.2.10",
-    "rebass": "^4.0.7"
+    "rebass": "^4.0.7",
+    "stream-browserify": "^3.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.10.2",
     "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-backgrounds": "^5.3.19",
-    "@storybook/addon-links": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
     "@storybook/react": "^5.3.19",
-    "autoprefixer": "^10.4.0",
```

Comment:
PostCSS requires Autoprefixer [as seen here](https://tailwindcss.com/docs/guides/gatsby), so I don't think we should remove it 😃

Reply:
My bad. Added again :)